_test/cli7.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/http/httptest"
+)
+
+type T struct {
+	http.ResponseWriter
+}
+
+type mw1 struct {
+	next http.Handler
```

Comment:
I think this field can be removed as it is not used, and does not bring anything to the table
```suggestion
```